m3.yml

```diff
@@ -127,6 +128,10 @@ properties:
       minimum: 0
       maximum: 100
     index_documentation: "searchable, displayable, creator facet"
+    indexing:
+      - "_tesim" # stored searchable
```

Comment:
tying these to blacklight style suffixes seems fragile, to me.

could we define a small indexing vocabulary (even just the text values you have in comments seem workable), and let applications map them as appropriate?

Reply:
Yes, that makes sense. Will work on it soon!